runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -132,9 +136,21 @@ private void setNativeContent(long newNativeContent) {
 
         assert mNativeContent == 0 && mCleanupReference == null && mContentViewCore == null;
 
+        boolean animated;
+        //If app developers want to assign the CompositingSurfaceType for a single xwalkview,
+        //"animatable" attribute of this namespace must be defined in activity.xml, while an attrs.xml
+        //that to define the type of "animatable" is also needed under res/values/
+        String namespace="http://schemas.android.com/apk/res-auto";
+
         // Initialize ContentViewRenderView
-        boolean animated = XWalkPreferencesInternal.getValue(
-                XWalkPreferencesInternal.ANIMATABLE_XWALK_VIEW);
+        if (attrs == null) {
+            Log.d(TAG, "Attributeset is null, get animated value from XWalkPreferences.");
+            animated = XWalkPreferencesInternal.getValue(
+                    XWalkPreferencesInternal.ANIMATABLE_XWALK_VIEW);
+        } else {
+            Log.d(TAG, "Config animated value by application xml.");
+            animated = attrs.getAttributeBooleanValue(namespace, "animatable", false);
+        }
```

Comment:
Removde above log, and add log, let user know he uses TextureView or SurfaceView.